packages/fxa-payments-server/src/index.tsx

```diff
@@ -58,7 +58,7 @@ async function getVerifiedAccessToken({
 
   try {
     const result = await fetch(
-      `${config.OAUTH_API_ROOT}/verify`,
+      `${config.servers.oauth.url}/v1/verify`,
```

Comment:
I ended up leaving the version numbers (`/v1`) out of the server config (it doesn't mean anything to ops), and then putting it right where the URLs are used. In a sense this fetch is coded to use the v1 protocol, so it seems somewhat correct to me that it's encoded here.

Reply:
Yeah, I think this makes sense.